code/API_definitions/Call_Forwarding_Signal.yaml

```diff
@@ -147,9 +141,14 @@ info:
     (FAQs will be added in a later version of the documentation)
     # Release Notes
     First draft for the CFS API.
```

Comment:
I don't think that release notes belong here. They would be difficult to maintain.

Reply:
I agree. 

They where requested here: https://github.com/camaraproject/Commonalities/blob/main/documentation/API-DocumentationTemplate.md

Currently we have changelog and release management, I think we can remove them from the YAML